backend/dataall/api/Objects/Vote/resolvers.py

```diff
@@ -49,3 +51,6 @@ def get_vote(context: Context, source, targetUri: str = None, targetType: str =
             data={'targetType': targetType},
             check_perm=True,
         )
+
+
+add_vote_type("dashboard", DashboardIndexer)
```

Comment:
I'd leave here a TODO note to move this to the Dashboard module as soon as it gets created

Reply:
Sure! Why not :)